packages/docs/programmable/mcp-server.md

```
@@ -66,7 +132,7 @@ Add to your MCP config (for example `.cursor/mcp.json`):
 }
 ```
 
-Or run from source without installing:
+Run from source without installing:
 
 ::: code-group
 ```json [Bun]
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)